flight/tests/misc_math/unittest.cpp

```diff
@@ -305,23 +305,24 @@ TEST_F(LinearInterpolate, ThrottleCurve1to1) {
   float const range_max = 1.0f;
   uint8_t const curve_numpts = 5;
   float const curve[5] = { 0.0f, 0.25f, 0.5f, 0.75f, 1.0f };
+  float eps = 0.000001f;
```

Comment:
This could probably be several orders of magnitude more strict, but 1e-6 is fine.